### PR TITLE
mcomix: update to 2.2.1.

### PR DESCRIPTION
**Pull request by hazen2215**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-musl)

### CODE REVIEW

File: srcpkgs/mcomix/template [hunk 1]
```diff
@@ -9,5 +9,6 @@ short_desc="GTK+ comic book viewer"
 maintainer="Leah Neukirchen <leah@vuxu.org>"
 license="GPL-2.0-only"
 homepage="http://sourceforge.net/projects/mcomix/"
```
Comment (classabbyamp) [inline srcpkgs/mcomix/template:11]: https

Comment (classabbyamp) [reply srcpkgs/mcomix/template:11]: also add changelog: https://sourceforge.net/p/mcomix/news/

**Comment by hazen2215** [comment]
Applied suggestion.

**Comment by Chocimier** [comment]
"broken tests" doesn't explain anything. Template lacks `checkdepends="python3-pytest ${depends}"` and tarball lacks some files present in source control, primarily `test/__init__.py`, what could be reported and resolved upstream.

**Pull request merged**